controllers/app_controller.go

```diff
@@ -86,44 +87,45 @@ func (r *AppReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.R
 		return ctrl.Result{}, err
 	}
 
-	var idpService *idp.Service
```

Comment:
I have swapped the idpservice and authservice orders bedcause identity provider setup can take a while during which the auth configmap doesn't exist and auth-bundle can not be deployed